index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <header id="header">
+      <img
+        id="header-img"
+        src="https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.logolynx.com%2Ftopic%2Frandom&psig=AOvVaw2VMZ1b3yO3Uin3J3lWBQ6N&ust=1653655505337000&source=images&cd=vfe&ved=0CAwQjRxqFwoTCMjdhPqY_fcCFQAAAAAdAAAAABAO"
+        alt=""
+      />
+      <nav id="nav-bar">
+        <ul class="nav-links">
+          <li class="nav-link"><a href="#header-img">Home</a></li>
+          <li class="nav-link"><a href="#header">About Us</a></li>
+          <li class="nav-link"><a href="#nav-bar">Contact Us</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <video id="video" src="https://www.youtube.com/watch?v=WILNIXZr2oc"></video>
+
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+      <input type="email" id="email" placeholder="Enter your email address" />
+      <input type="submit" id="submit" />
+    </form>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script src="js/index.js"></script>
+  </body>
+</html>
```

Comment:
Please add an extra line here. If the last line in a file doesn't end with a newline then addition of next line affects two lines instead of one. You can read more [here](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3)

Reply:
[Optional] I think it would be better to use a couple of emojis and/or gifs in the review to make the code review more human-friendly👽🤖